python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -315,6 +315,17 @@ def _get_entity_details(self, idx):
         if not idx.isValid():
             return {}
 
+        # first, ensure that all child data has been loaded
+        def __fetch_more_r(model_index):
```

Comment:
it's a super custom method meant for this purpose only. But i can move it out if you feel that improves readability!